src/include/execution/sql/functions/string_functions.h

```diff
@@ -122,6 +122,11 @@ class EXPORT StringFunctions {
    * Converts the string to upper case
    */
   static void Upper(exec::ExecutionContext *ctx, StringVal *result, const StringVal &str);
+
+  /**
+   * Gets the version of the database
+   */
+  static void Version(exec::ExecutionContext *ctx, StringVal *result);
```

Comment:
This is not really a string function. Make a new file called `include/execution/sql/functions/system_functions.h`

Reply:
Updated.